### PR TITLE
Added a new mentor(Priyanshu Khandelwal) and his photo.

### PR DESCRIPTION
**Pull request by rathindra3376**
- [yes ] Included a Preview link and screenshot showing after and before the changes.
- [yes ] Included a description of the change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link]
(https://github.com/rathindra3376/gci18.fossasia.org/blob/master/_data/mentors.yml)
(https://github.com/rathindra3376/gci18.fossasia.org/tree/master/img/mentors)

- Fixes #571 

## Description / Changes
I added his photo by checking him in the task and website.

### After:

- - - - - - - - - - - -

### CODE REVIEW

**Comment by rathindra3376** [comment]
What confilts?

File: _data/students.yml [hunk 1]
```diff
@@ -179,4 +179,8 @@
 
 - name: Hồ Đức Hiếu
   github: hoduchieu01
-  image: hoduchieu01.png
\ No newline at end of file
+  image: hoduchieu01.png
+
```
Comment (pythongiant) [inline _data/students.yml]: please only work at one issue in one PR

File: js/thankyou.js [hunk 2]
```diff
@@ -31,7 +31,7 @@
 'സംഭാവന ചെയ്തത്തിന് നന്ദി',
 'Takk fyrir þitt framlag!',
 'Tibi gratias ago pro tui contributions!'
-
+'Salamat sa iyong mga kontribusyon!'
```
Comment (pythongiant) [inline js/thankyou.js]: Please work at only one issue per PR

**Comment by pythongiant** [comment]
Give a correct preview link

**Pull request closed without merging**